### PR TITLE
Add Manage Team feature

### PR DESCRIPTION
**Pull request by bhargavaparoksham**
Add Ability to Add / Remove members

Deploy

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

File: packages/round-manager/src/features/round/ViewManageTeam.tsx [hunk 1]
```diff
@@ -43,27 +88,159 @@ export default function ViewManageTeam(props: { round: Round | undefined }) {
     return filterRoles(sortedRoles);
   }, [sortedRoles]);
 
+  const isTeamMembersLoading = indexingStatus == ProgressStatus.IN_PROGRESS;
+
+  console.log("TEAM", isTeamMembersLoading)
+  console.log("CON", contractUpdatingStatus)
+  console.log("INDE", indexingStatus)
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/round/ViewManageTeam.tsx:95]:
```suggestion
```

File: packages/round-manager/src/features/round/ViewManageTeam.tsx [hunk 2]
```diff
@@ -43,27 +88,159 @@ export default function ViewManageTeam(props: { round: Round | undefined }) {
     return filterRoles(sortedRoles);
   }, [sortedRoles]);
 
+  const isTeamMembersLoading = indexingStatus == ProgressStatus.IN_PROGRESS;
+
+  console.log("TEAM", isTeamMembersLoading)
+  console.log("CON", contractUpdatingStatus)
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/round/ViewManageTeam.tsx:94]:
```suggestion
```

File: packages/round-manager/src/features/round/ViewManageTeam.tsx [hunk 3]
```diff
@@ -43,27 +88,159 @@ export default function ViewManageTeam(props: { round: Round | undefined }) {
     return filterRoles(sortedRoles);
   }, [sortedRoles]);
 
+  const isTeamMembersLoading = indexingStatus == ProgressStatus.IN_PROGRESS;
+
+  console.log("TEAM", isTeamMembersLoading)
```
Comment (thelostone-mc) [inline packages/round-manager/src/features/round/ViewManageTeam.tsx:93]:
```suggestion
```

**Review by thelostone-mc — COMMENTED** [review]
meh

**Review by melissa-neira — APPROVED** [review]
everything looks and works great! my one call out is that it is annoying that after you add a member, the page refreshes and you are taken to the grant applications tab. is there any way for the page to refresh to the same tab (manage team)?

**Review by melissa-neira — CHANGES_REQUESTED** [review]
oops, meant to request changes!

everything looks and works great! my one call out is that it is annoying that after you add a member, the page refreshes and you are taken to the grant applications tab. is there any way for the page to refresh to the same tab (manage team)?

**Pull request merged**